lib/t/accounts.ex

```diff
@@ -444,6 +444,20 @@ defmodule T.Accounts do
     get_user_by_session_token_and_update_version(token, nil, context)
   end
 
+  @spec list_app_versions(Ecto.Bigflake.UUID.t()) :: [String.t()]
+  def list_app_versions(user_id) do
+    UserToken |> where(user_id: ^user_id) |> select([t], t.version) |> Repo.all()
+  end
+
+  @spec has_previous_version?(Ecto.Bigflake.UUID.t()) :: bool
+  def has_previous_version?(user_id) do
```

Comment:
@ruslandoga is this and the command above used anywhere?
couldn't find. can I remove them?

Reply:
Not used.

I wanted to use it for stats, but now I see that we can just read it from DB. I'd be very thankful if you remove them. 

Thank you.